library.json

```diff
@@ -1,11 +1,7 @@
 {
-  "name": "RCSwitch",
-  "description": "Use your Arduino or Raspberry Pi to operate remote radio controlled devices",
+  "name": "rc-switch",
+  "description": "Arduino lib to operate 433/315Mhz devices like power outlet sockets",
```

Comment:
Need to ask owner of this repository. I copied it from https://github.com/sui77/rc-switch